google/generativeai/types/retriever_types.py

```diff
@@ -522,6 +524,8 @@ class Document(abc.ABC):
     name: str
     display_name: str
     custom_metadata: list[CustomMetadata]
+    create_time: str
```

Comment:
You can create the proto from a string, but I the proto, and the `idecode_time` functions are converting them to Datetime objects.

https://github.com/google/generative-ai-python/blob/main/google/generativeai/types/model_types.py#L118-L128